### PR TITLE
Add tasks 15 to 17

### PR DESCRIPTION
**Pull request by MartinUzunov1707**
Added tasks 15, 16 and 17.

### CODE REVIEW

**Comment by Slavi15** [comment]
Task 15 and 16 are fine!
In Task 17, rather than using [ +/- 32 ] to convert between uppercase and lowercase letters, you should do (ch - 'A') + 'a' for conversion from uppercase to lowercase and respectively (ch - 'a') + 'A' for the latter conversion. Also, if you want to check if a char is capital letter, you could just do (ch >= 'A' && ch <= 'Z'). Try to segregate your code into more variables and functions, when we learn them. Do not forget that STL functions are forbidden!
To sum up, very good work, keep on going!

**Comment by Slavi15** [comment]
Also, please put the different tasks and their solutions into distinct cpp files the next time you make a pull request, i.e Task15.cpp, Task16.cpp, etc.!
